spring-cloud-sleuth-zipkin/src/test/java/org/springframework/cloud/sleuth/zipkin/HttpZipkinSpanReporterTest.java

```diff
@@ -20,22 +26,18 @@
 import zipkin.junit.HttpFailure;
 import zipkin.junit.ZipkinRule;
 
-import static java.util.Arrays.asList;
```

Comment:
revert re-org of imports as well all of the other formatting changes to this file